go/runtime/host/sgx/epid.go

```diff
@@ -68,13 +69,32 @@ func (ep *teeStateEPID) Update(ctx context.Context, sp *sgxProvisioner, conn pro
 		return nil, fmt.Errorf("error while getting quote: %w", err)
 	}
 
+	// Get current quote policy from the consensus layer.
+	var quotePolicy *cmnIAS.QuotePolicy
+	var policies *sgxQuote.Policy
+	policies, err = ep.getQuotePolicies(ctx, sp)
+	if err != nil {
+		return nil, err
+	}
+	if policies != nil {
+		quotePolicy = policies.IAS
+	}
+
 	evidence := ias.Evidence{
 		RuntimeID: ep.runtimeID,
 		Quote:     quote,
 		Nonce:     nonce,
 	}
+	if quotePolicy != nil {
+		evidence.MinTCBEvaluationDataNumber = quotePolicy.MinTCBEvaluationDataNumber
+	}
 
 	avrBundle, err := sp.ias.VerifyEvidence(ctx, &evidence)
+	if err != nil && quotePolicy.MinTCBEvaluationDataNumber > 0 {
```

Comment:
I would compare against the avr here instead of delegating the `MinTCBEvaluationDataNumber` check to the IAS proxy.